lnwallet/interface_test.go

```diff
@@ -3130,9 +3130,18 @@ func runTests(t *testing.T, walletDriver *lnwallet.WalletDriver,
 	// Execute every test, clearing possibly mutated
 	// wallet state after each step.
 	for _, walletTest := range walletTests {
+
+		walletTest := walletTest
+
 		testName := fmt.Sprintf("%v/%v:%v", walletType, backEnd,
 			walletTest.name)
 		success := t.Run(testName, func(t *testing.T) {
+			if backEnd == "neutrino" &&
+				strings.Contains(walletTest.name, "dual funder") {
```

Comment:
Nit: just add backend check in the test case itself?

Reply:
AFAICT, the test doesn't know which backend is being used. The config for `btcwallet` is a private variable so the test can't access that to perform a type assertion, I tried that route before this one.